src/Commands/CacheToolsCommands.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Drupal\cache_tools\Commands;
+
+use Drush\Commands\DrushCommands;
+
+/**
+ * Provides commands to purge persistent cache objects.
+ */
+class CacheToolsCommands extends DrushCommands {
+
+  /**
+   * Purges data from persistent cache tables.
+   *
+   * @command cache:purge
+   * @aliases cache-nuke,cache-tools-purge
+   */
+  public function toolsPurge() {
+    $this->output()->writeln(dt('Preparing to truncate persistent cache tables... '));
+
+    // Clear all caches to purge expired objects first.
+    drupal_flush_all_caches();
```

Comment:
The drush9 is throwing the error after reaching here
`drush9 cache:purge
Preparing to truncate persistent cache tables... 
 [error]  \Drupal::$container is not initialized yet. \Drupal::setContainer() must be called with a real container. `

Is there any additional makeup required to load the container?